.github/workflows/maven-verify.yml

```diff
@@ -152,6 +152,12 @@ on:
         default: false
         type: boolean
 
+      install-gpg:
+        description: Ensure all build images have GnuPG installed
```

Comment:
Description is somewhat in disparity with what it actually does:
* installs gpg4win (on Windows)
* tests is gpg install (on all OSes)

Reply:
After last commit this is actually fine: on Win it installs (as others does not need it), and on all OS runs `gpg --version`.